### DISCUSSION TITLE
set Relation negation

### DISCUSSION THREAD

**Discussion by hamaney** [typst/typst#4362] [Q&A] [answered]
How to make this R that represents Relation. I was able to do the R in this style by doing cal(R) but I am not sure how to mark it with not.

<img width="133" alt="image" src="https://github.com/typst/typst/assets/11434409/4c42b058-3f64-45a3-8bea-33b80c219444">

**Comment by MDLC01** [comment, marked as answer]
Take a look at the [`cancel`](https://typst.app/docs/reference/math/cancel/) function.

The following should achieve the desired result. It also uses the [`class`](https://typst.app/docs/reference/math/class/) function to help with spacing.

```typ
#let rel = $class("relation", cancel(cal(R)))$
$x rel y$
```
